docs/src/pages/components/buttons/buttons.md

```diff
@@ -3,7 +3,7 @@ title: Button React component
 components: Button, IconButton, ButtonBase
 ---
 
-# Button
+# Buttons
```

Comment:
We are moving to singular everywhere

```suggestion
# Button
```

Reply:
Gotcha. I'll move the plural ~

Reply:
Thanks, at least for the title of the components :)